Source/WebKit/UIProcess/API/Cocoa/_WKAttachment.h

```diff
@@ -45,6 +45,7 @@ WK_CLASS_AVAILABLE(macos(10.14), ios(12.0))
 @property (nonatomic, readonly, nullable) NSString *filePath;
 @property (nonatomic, readonly, nullable) NSData *data;
 @property (nonatomic, readonly, nullable) NSFileWrapper *fileWrapper;
+@property (nonatomic, readonly) BOOL preserveFidelity;
```

Comment:
Nit - maybe `-preservesFidelity` or `-shouldPreserveFidelity`, to make it clear that it's a property? (As opposed to an action method, like `-[_WKAttachment preserveFidelity]`).

I think we might _technically_ need availability declarations too.

Reply:
Went with `shouldPreserveFidelity` and added availability declarations.